ci/install-test-deps.sh

```diff
@@ -16,7 +16,7 @@ case "$OS" in
         yum install -y expect json-glib-devel libtasn1-devel net-tools python3 socat
         ;;
 
-    # NOTE: ubuntu:20.04 uses libtss2-dev provided through the package
+    # NOTE: ubuntu:20.04+ uses libtss2-dev provided through the package
```

Comment:
Are you sure this is true for 22.04?

Reply:
Ah, no.  In fact, it looks likely that it might have a TCTI module given the libtss2-dev's [list of files in 22.04](https://packages.ubuntu.com/jammy/amd64/libtss2-dev/filelist).